README.md

```diff
@@ -19,27 +19,43 @@ example pipeline.
 Getting started - prerequisites
 ---
 
-You can use `podman` or `docker`. These examples use `podman`. Note the
-container must be privileged, as the build process uses container functionality
-itself - we're using [recursive containers](https://github.com/projectatomic/bubblewrap/issues/284).
-
-Secondly, in order to build VM images, the container must have access to
-`/dev/kvm`.  If you're running this in a VM, you must enable
+You can use `podman` or `docker`. These examples use `podman`. If using
+the latter, you may run it fully unprivileged. When using `docker`, the
+container must be privileged, as the build process uses container
+functionality itself - we're using
+[recursive containers](https://github.com/projectatomic/bubblewrap/issues/284).
+
+Secondly, the container must have access to `/dev/kvm`.  If you're
+running this in a VM, you must enable
 [nested virt](https://docs.fedoraproject.org/en-US/quick-docs/using-nested-virtualization-in-kvm/).
 See also [GCE nested virt](https://cloud.google.com/compute/docs/instances/enable-nested-virtualization-vm-instances).
 
 Setup
 ---
 
 Here we store data in `/srv/coreos` on our host system.  You can choose
-any directory you like.  You should run these commands as `root`.
+any directory you like.
+
+If running as root (or with `sudo` access):
 
 ```
 $ mkdir /srv/coreos
 $ cd /srv/coreos
-$ alias coreos-assembler='podman run --rm --net=host -ti --privileged --userns=host -v $(pwd):/srv --workdir /srv quay.io/coreos-assembler/coreos-assembler'
+$ alias coreos-assembler='sudo podman run --rm --net=host -ti --privileged --userns=host -v $(pwd):/srv --workdir /srv quay.io/coreos-assembler/coreos-assembler'
+```
+
+If running rootless:
+
+```
+$ mkdir -m 777 /srv/coreos
```

Comment:
World writable directory without `+t`?  Not a good idea to encourage, why are we doing this?

Reply:
OK, fixed this up now! :arrow_down: The missing piece was using `--user=root` so that we run as root in the user namespace, but `$UID` on the host. Before, we were running as `builder`, which mapped to some other uid, which is why I was using `-m 777`.